src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractTypeAwareCheck.java

```diff
@@ -240,7 +240,9 @@ private ClassResolver getClassResolver() {
         try {
             clazz = getClassResolver().resolve(resolvableClassName, className);
         }
-        catch (final ClassNotFoundException ignored) {
+        // -@cs[IllegalCatch] We need to catch all exceptions.
+        // See https://github.com/checkstyle/checkstyle/issues/6606
```

Comment:
no need to reference issue, it is already in git history.

lets make comment `exception type is not predictable`

Reply:
Done.